request.js

```diff
@@ -459,6 +464,10 @@ TChannelRequest.prototype.shouldRetryError = function shouldRetryError(err) {
         return false;
     }
 
+    if (self.channel.currentRetryRatio() > self.maxRetryRatio) {
+        return false;
```

Comment:
Aha, currently tchannel will emit the "last failure" when it cannot retry again.

So the way to have visibility without adding new stats is to look at the retries graph and see if it flat lines at the 10% cap.

If you look at success, error & retries then in theory you should see something like :

 - 100% success,
 - 100% success, 2% retries,
 - 99% success, 5% retries, 1% error
 - 98% success, 10% retries, 2% error,
 - 88% success, 10% retries, 12% errors

However we can emit new metrics for retry bailout. That's probably a good idea.